omise/config.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <module>
-    <name>omise</name>
-    <displayName><![CDATA[Omise]]></displayName>
-    <version><![CDATA[1.5]]></version>
-    <description><![CDATA[]]></description>
-    <author><![CDATA[Omise]]></author>
-    <tab><![CDATA[payments_gateways]]></tab>
-	<confirmUninstall><![CDATA[Are you sure you want to uninstall Omise module?]]></confirmUninstall>
-    <is_configurable>1</is_configurable>
-    <need_instance>0</need_instance>
+	<name>omise</name>
+	<displayName><![CDATA[Omise]]></displayName>
+	<version><![CDATA[1.6]]></version>
+	<description><![CDATA[]]></description>
+	<author><![CDATA[Omise]]></author>
+	<tab><![CDATA[payments_gateways]]></tab>
+	<confirmUninstall><![CDATA[Are you sure you want to uninstall the Omise module?]]></confirmUninstall>
```

Comment:
@jonrandy should we convert this line to `space` instead?

Reply:
Can do. I didn't consciously change it. Must have been my editor

Reply:
I changed it, and PrestaShop itself changed it back. I'll leave it